test/spec.js

```diff
@@ -0,0 +1,48 @@
+describe('todoApp homepage', function() {
+  
+  beforeEach(function() {
+    browser.get('http://localhost:8080');
+  });
+
+  it('should have a title', function() {
+    expect(browser.getTitle()).toEqual('To-Do List');
+  });
+
+  it('should have an todo form', function() {
+    expect(browser.isElementPresent(by.id('todoForm'))).toBe(true);
+  });
+  
+  it("a user can add a todo item", function() {
+    element(by.model('newTodo')).sendKeys('Task 1');
+    element(by.id('addTodo')).click();
+    expect(element(by.repeater('todo in todos')).getText()).toContain("Task 1");
+  });
+
+  it("tracks the number of todo items", function() {
+    element(by.model('newTodo')).sendKeys('Task 1');
```

Comment:
Same here, you could use a helper function so you don't repeat yourself when creating new tasks.